Server/controllers/monitorController.js

```
@@ -390,18 +390,18 @@ class MonitorController {
 
 			await Promise.all(
 				notifications &&
-					notifications.map(async (notification) => {
-						notification.monitorId = editedMonitor._id;
-						await this.db.createNotification(notification);
-					})
+				notifications.map(async (notification) => {
+					notification.monitorId = editedMonitor._id;
+					await this.db.createNotification(notification);
+				})
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)